server/libmain/libmain.go

```diff
@@ -423,10 +423,10 @@ func StartAndRunServices(env *real_environment.RealEnv) {
 
 	mux := env.GetMux()
 	// Register all of our HTTP handlers on the default mux.
-	mux.Handle("/", interceptors.WrapExternalHandler(env, staticFileServer))
+	mux.Handle("/", interceptors.WrapAuthenticatedExternalHandler(env, staticFileServer))
```

Comment:
this doesn't break static file serving or anything?

Reply:
I also tested this locally - I think it shouldn't break anything since if the user is not authenticated, the middleware will just put an auth error into the context, but still proceed - and the static file server doesn't check for authentication afaik.

(Also, FWIW, this codepath should be be getting exercised in all of the webdriver tests.)